src/data/cloud.ts

```diff
@@ -78,9 +80,9 @@ export const fetchCloudSubscriptionInfo = (hass: HomeAssistant) =>
 export const updateCloudPref = (
   hass: HomeAssistant,
   prefs: {
-    google_enabled?: boolean;
-    alexa_enabled?: boolean;
-    google_allow_unlock?: boolean;
+    google_enabled?: CloudPreferences["google_enabled"];
+    alexa_enabled?: CloudPreferences["alexa_enabled"];
```

Comment:
Shouldn't these be required if their `CloudPreferences` counter-parts are required?

Reply:
We only submit the preferences that we want to change when updating the prefs.